terraform/paas/variables.tf

```diff
@@ -37,6 +37,10 @@ variable "paas_space" {
   default = "sandbox"
 }
 
+variable "paas_monitoring_app" {
+  default = "prometheus-dev-get-into-teaching"
```

Comment:
If we always expect to pass a value, we can remove the default

Reply:
changed